### DISCUSSION TITLE
想请问一下c++版本的paddleocr会比python版本的推理速度更快嘛？

### DISCUSSION THREAD

**Discussion by shumeirao** [PaddlePaddle/PaddleOCR#12403] [Q&A]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：
- 版本号/Version：Paddle：  PaddleOCR： 问题相关组件/Related components：
- 运行指令/Command Code：
- 完整报错/Complete Error Message：

我们提供了AceIssueSolver来帮助你解答问题，你是否想要它来解答(请填写yes/no)?/We provide AceIssueSolver to solve issues, do you want it? (Please write yes/no):

请尽量不要包含图片在问题中/Please try to not include the image in the issue.

**Comment by SWHL** [comment]
当然会啊。c++版比python版快在图像的前后处理操作上。推理引擎是一样的。
